src/conftest.py

```diff
@@ -14,6 +14,19 @@
 from organizations.models import Organization
 
 
+@pytest.fixture(scope="session")
+def django_db_setup(django_db_setup, django_db_blocker):
+    with django_db_blocker.unblock():
+        with connection.cursor() as cursor:
+            cursor.execute(
+                """
+                CREATE TEXT SEARCH CONFIGURATION french_unaccent( COPY = french );
+                ALTER TEXT SEARCH CONFIGURATION french_unaccent
+                ALTER MAPPING FOR hword, hword_part, word WITH unaccent, french_stem;
+                """
+            )
+
+
```

Comment:
Je pense qu'il faut faire un rebase de master ou supprimer cet ajout (comme on vient de le supprimer dans la [PR1026](https://github.com/MTES-MCT/aides-territoires/pull/1026/files) :)

Reply:
Si j'enlève ça mes tests ne passent plus en local :/

Reply:
Il faut que tu te connectes à ta db et que tu passes la commande `psql -d template1 -c 'CREATE TEXT SEARCH CONFIGURATION french_unaccent( COPY = french ); ALTER TEXT SEARCH CONFIGURATION french_unaccent ALTER MAPPING FOR hword, hword_part, word WITH unaccent, french_stem;' -U postgres`